code/modules/mob/living/carbon/human/species.dm

```diff
@@ -223,6 +223,9 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	/// Do we try to prevent reset_perspective() from working? Useful for Dullahans to stop perspective changes when they're looking through their head.
 	var/prevent_perspective_change = FALSE
 
+	///Was the species forcefully changed from its original type?
+	var/forced_change = FALSE
+
```

Comment:
please note that "forcefully changed" only applies during job assignment, otherwise people may think it's used in other cases (like purrbation or something)

Reply:
Right, ill rename it